configs/mask2former/mask2former_r50_lsj_8x2_50e_coco_ins.py

```diff
@@ -0,0 +1,234 @@
+_base_ = [
```

Comment:
Is it same as the config of instance segmentation on https://github.com/facebookresearch/Mask2Former

Reply:
There is no need to write so much code to configure, you can inherit the config file of panoptic segmentation and modify some options

Reply:
Thanks.  I will check the original implementation and make sure they have identical settings then inherit to reduce the amount of code.

Reply:
Checked the original implementation and they use the same settings for instance segmentation and panoptic segmentation.  Compared their settings to ours and did not see any differences.  

Reworked the configs so the instance segmentation configs inherit from the primitive panoptic segmentation config.  The data pipeline and data definitions were fully redefined because inheriting from configs/_base_/datasets/coco_instance.py caused the builder to find duplicate keys.